.gitignore

```diff
@@ -73,3 +73,6 @@ build/
 !**/ios/**/default.mode2v3
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
+
+
+.gitignore
```

Comment:
You have git ignore in the git ignore file? 😄
Is this to avoid adding the git ignore from the example?

Reply:
derp, it was supposed to be the locks